engine/src/execution_graph/logic_controllers/LogicalProject.cpp

```diff
@@ -440,7 +453,23 @@ std::unique_ptr<cudf::column> evaluate_string_functions(const cudf::table_view &
         
         computed_col = cudf::strings::to_upper(column);
         break;         
-    }        
+    }
+    case operator_type::BLZ_STR_TRIM:
+    {
+        assert(arg_tokens.size() == 3);
+        RAL_EXPECTS(!is_literal(arg_tokens[2]), "TRIM operator not supported for literals");
```

Comment:
Why is the trim operator for literals not implemented? I know it doesn't feel useful but it comes in handy when users do thing like generate SQL using UI's or tools that build sql statements.

Reply:
> Why is the trim operator for literals not implemented?

 No concrete reason, was loosely following the pattern for other string functions. 
Happy to implement trim for literals as well. Since it's a literal would it make sense to implement it on the host rather than going via libcudf? I see some string utils which can potentially be overloaded to trim arbitrary strings from a literal instead of whitespaces? 
https://github.com/BlazingDB/blazingsql/blob/7e41958bc6708473925ef73216bfbc45de8c3ee1/io/src/Util/StringUtil.cpp#L434-L446